seyik/controllers/likeController.js

```diff
@@ -0,0 +1,25 @@
+//controller/likeController.js
+
+const likeService = require("../services/likeService");
+
+const like = async (req, res) => {
+  try {
+    const { userId, postId } = req.body;
+
+    if (!userId || !postId) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await likeService.like(userId, postId);
+    return res.status(201).json({
+      message: "LIKE_SUCCESS",
+    });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 400).json({ message: err.message });
```

Comment:
|| 뒤의 상태코드는 err.statusCode가 정의 되지 않았을 때 사용할 코드인데, err.statusCode가 정의되지 않았다는 것은 예외처리를 제대로 하지 않았다는 것이므로 500이 적절합니다.

이 부분뿐만 아니고 다른 부분들도 전부 고쳐주세요.
```suggestion
    return res.status(err.statusCode || 500).json({ message: err.message });
```